addons/mail/static/tests/owl/component/chat_window_manager_tests.js

```diff
@@ -550,8 +550,8 @@ QUnit.test('chat window: state conservation on toggle home menu', async function
     // Set a scroll position to chat window
     document.querySelector(`.o_Thread_messageList`).scrollTop = 142;
     // Set html content of the composer of the chat window
-    let composerTextInputEditable = document.querySelector(`.o_ComposerTextInput_editable`);
-    composerTextInputEditable.focus();
+    let composerTextInputTextArea = document.querySelector(`.o_ComposerTextInput_textarea`);
+    composerTextInputTextArea.focus();
     document.execCommand('insertText', false, 'XDU for the win !');
```

Comment:
very honorable of you to leave @xdu-odoo alone in this file

Reply:
XDU for the win !

Reply:
....

Reply:
Thanks for the reminding. I'll remove it.